src/pages/home/report/ReportActionItem.js

```diff
@@ -232,6 +234,12 @@ class ReportActionItem extends Component {
 
         const reactions = _.get(this.props, ['action', 'message', 0, 'reactions'], []);
         const hasReactions = reactions.length > 0;
+        const shouldDisplayThreadReplies = lodashGet(this.props.action, 'childReportID', 0) !== 0 && lodashGet(this.props.action, 'childCommenterCount', 0) > 0;
```

Comment:
i don't see a reason why checking childReportID is important here. Just looking at `childCommenterCount` should suffice no?